models/text_recognition_crnn/README.md

```diff
@@ -2,11 +2,23 @@
 
 An End-to-End Trainable Neural Network for Image-based Sequence Recognition and Its Application to Scene Text Recognition
 
+Results of accuracy evaluation with [tools/eval](../../tools/eval).
+
+| Models | Accuracy(%) |
+| ------ | -------------- |
+| CRNN_EN | 81.66 |
+| CRNN_EN_FP16 | 82.01 |
+| CRNN_CH | 71.28 |
+| CRNN_CH_FP16 | 78.63 |
+\*: 'FP16' stands for 'model quantized into FP16'.
```

Comment:
We should carry experiment on more OCR datasets. Then, I think we can merge it. Please add info about the test dataset.

Reply:
@Charles-258, friendly reminder